generators/app/templates/page_objects/home.page.ts

```diff
@@ -1,6 +0,0 @@
-import { browser, $} from 'protractor'
```

Comment:
Keep this file in original project. This is just example how to built pageobjects